### PR TITLE
Admin link

### LINKED ISSUE #7806: Admin Title
The title of the site, in the upper left of the admin, is not styled like previously.

Before:
image
Now:
image

Could it be possible to find also a way to add a link to /admin? 
By clicking the logo for example?

### PR DESCRIPTION
**Pull request by agriffard**
Fixes #7806

Separate the upper left infos of the admin in two links:
- The logo is a link to the admin.
- The title is a link to the site.

### CODE REVIEW

**Comment by hishamco** [comment]
I like the changes based on the screenshot, hope @Skrypt like it too

**Comment by Skrypt** [comment]
image

**Review by Skrypt — COMMENTED** [review]
image

There is an extra margin there that looks odd between the user button and the 2 others. One of them is missing cursor:hand which can be fixed by adding role="button" on the anchor. The tenant name now has a blue color which I think shouldn't. User button should be the last one.

See Github header :  

image

**Comment by agriffard** [comment]
ok, UserMenu moved to navbar-collapse => margin removed.

File: src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml [hunk 1]
```diff
@@ -30,30 +31,42 @@
...
+                                }
+                                else
+                                {
+                                    <i class="fas fa-moon"></i>
+                                }
+                            </a>
+                            }
+                        </li>
+                        <li class="nav-item">
+                            <a href="@Url.Content("~/")" class="nav-link" target="_blank" data-toggle="tooltip" title="@T["Visite Site"]"><i class="fa fa-external-link"></i></a>
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml:65]:
```suggestion
                            <a href="@Url.Content("~/")" class="nav-link" target="_blank" data-toggle="tooltip" title="@T["Visit Site"]"><i class="fa fa-external-link"></i></a>
```

File: src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml [hunk 2]
```diff
@@ -30,30 +31,42 @@
...
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse flex-grow-0" id="navbarNav">
+                    <ul class="navbar-nav">
+                        @await RenderSectionAsync("NavbarLinks", required: false)
+                        <li class="nav-item">@await DisplayAsync(await New.UserMenu())</li>
+                        <li class="nav-item">
+                            @if (adminSettings.DisplayDarkMode)
+                            {
+                            <a id="btn-darkmode" class="nav-link" data-toggle="tooltip" title="@(darkMode ? T["Light mode"] : T["Dark mode"])">
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml:52]:
```suggestion
                            <a id="btn-darkmode" class="nav-link" data-toggle="tooltip" title="@(darkMode ? T["Light mode"] : T["Dark mode"])" role="button">
```

Comment (Skrypt) [reply src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml]: A anchor without a href requires `role="button"` to be using `cursor: pointer`.

File: src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml [hunk 3]
```diff
@@ -1,15 +1,17 @@
 @inject IAuthorizationService AuthorizationService
-<form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline">
-    <div class="dropdown">
-        <button class="btn btn-sm btn-light dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            <span><i class="fa fa-user" aria-hidden="true"></i></span>&nbsp;<span>@User.Identity.Name</span>
-        </button>      
-        <div class="dropdown-menu dropdown-menu-right w-25">
+<div class="dropdown">
+    <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline">
+        <a class="nav-link dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml:4]: <a class="nav-link dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button">

Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml:4]:
```suggestion
        <a class="nav-link dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button">
```

File: src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
+            <span><i class="fa fa-user" aria-hidden="true"></i>
+        </a>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+            <h6 class="dropdown-header">@User.Identity.Name</h6>
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml:8]:
```suggestion
            <h6 class="dropdown-header">@T["Signed in as"]<span class="ml-1 font-weight-bold">@User.Identity.Name</span></h6>
```

Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml:8]: Not sure for this one.

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml:8]: Not necessary but it is consistent with what Github does here :  

image

**Comment by agriffard** [comment]
image

**Comment by agriffard** [comment]
With `Signed in as`:

image

**Comment by Skrypt** [comment]
The only thing "buggy" is the tooltip on the dark/light mode button. We would need to be able to create translations for javascript vars.

**Comment by agriffard** [comment]
> The only thing "buggy" is the tooltip on the dark/light mode button. We would need to be able to create translations for javascript.

Why does it seems buggy according to you?

**Comment by Skrypt** [comment]
When I click on it the Tooltip title doesn't toggle.

**Pull request merged**